http/api/endpoint_test.go

```diff
@@ -63,10 +63,14 @@ func (tr testRequest) make() (*http.Response, error) {
 
 func TestPublish(t *testing.T) {
 	chanID := "1"
-	contentType := "application/senml+json"
+	contentTypeSenML := "application/senml+json"
+	contentTypeCBOR := "application/senml+cbor"
```

Comment:
Looks like naming is not consistent. If we call upper SenML (should be maybe `SenMLJSON`), this one is also SenML (just that it is CBOR encoded). So in the upper we use semantic representation (SenML) and in the second we use encoding/serialization (although semantic structures are still the same).